### PR TITLE
making the website more up to date

### PR DESCRIPTION
**Pull request by popcornell**
I added some recent challenges and workshop (2024 and 2023) and also made the same changes as done in the ISCA website

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -29,25 +29,32 @@ Welcome to the webpage of ISCA's Special Interest Group (SIG) on Robust Speech P
 * [1st CHiME Speech Separation and Recognition Challenge](https://www.isca-speech.org/archive/chime_2011/index.html)
 
 # SIG events
```
Comment (sw005320) [inline index.md:31]: SIG events should be endorsed by ISCA via RoSP SIG.
Does SASB correspond to that?
If not, we can move it to the related activities.

Comment (sw005320) [inline index.md:31]: To avoid the confusion, maybe we can clarify it?
```suggestion
# RoSP SIG events endorsed by ISCA
```

Comment (popcornell) [reply index.md]: I asked, it is not. Moving it !

**Comment by popcornell** [comment]
@leibny is this fine ?

**Pull request merged**